frontend/src/components/LogIn/login.css

```diff
@@ -118,3 +128,34 @@ img {
     margin-right: 5%;
     margin-bottom: 0;
 }
+
+.errorMessage {
+    font-weight: 250;
+    font-size: 12px;
+    margin-top: -10px;
+    text-align: left;
+    padding-left: 20%;
+    padding-right: 20%;
+    color: red;
+}
+
+.longErrorMessage {
+    font-weight: 250;
+    font-size: 12px;
+    margin-top: 5px;
+    margin-bottom: 15px;
+    margin-left: 20%;
+
+    text-align: center;
+    color: red;
+    background-color: rgba(255, 177, 177, 0.597);
+    max-width: 60%;
+    padding: 1% 1% 1% 1%;
+    border-radius: 15px;
+}
+.reset {
+    text-decoration: underline;
+}
+.reset:hover{
```

Comment:
Same kind of comments as in your Other PR. 

Lets look into modules as well as making sure we format our code consistently.